src/ExtractImportGraph.elm

```diff
@@ -10,62 +10,74 @@ import Dict exposing (Dict)
 
-## Success
 
-    a =
-        "REPLACEME example to replace"
+## Try it out
 
+You can try this rule out by running the following commands.
 
-## When (not) to enable this rule
+The easiest way to try out is to run the following command (requires [`jq`](https://stedolan.github.io/jq/)):
 
-This rule is useful when REPLACEME.
-This rule is not useful when REPLACEME.
+```bash
```

Comment:
All three of these should probably point to `example` not `preview`, in keeping with [other `elm-review` packages](https://package.elm-lang.org/packages/jfmengels/elm-review-common/latest#try-it-out).

Reply:
If you intend to publish it, then yes. If you don't, then I'd keep it as `preview` otherwise it will not work for people.

Reply:
That's mostly on you as far as how you conceive of the extract rule ecosystem.  Is the expectation that extract rules won't be published?  That might make discoverability a challenge.

Reply:
Ah no, it's mostly on you since it's under your name :smile: since publishing it under your name will make people come to you mostly for fixes and improvements.

I expect some rules to stay unpublished as in more of a "you can try it out with `--template` and you can fork it to customize it", but I'm happy to see some rules published. Shall we publish an initial version of this package now?